hack/generated/pkg/reconcilers/azure_deployment_reconciler.go

```diff
@@ -110,6 +110,8 @@ func NewAzureDeploymentReconciler(
 }
 
 func (r *AzureDeploymentReconciler) CreateOrUpdate(ctx context.Context) (ctrl.Result, error) {
+	r.logObj("reconciling resource")
```

Comment:
Do we have the information here to log _which_ resource we're reconciling?

Reply:
It's automatically included as a variable pair on `r.log` already (`namespace` and `name`, as well as the specific controller type that is doing the logging). So everything that `r.log` logs has it automatically - which is nice because it keeps the code where we log short as it's just the "extra stuff" that has to be passed there.

Agree it does look a bit weird at first glance though.

Reply:
Ahh, good technique. Nice.